PlaybackLab/PictureInPictureSampleApp/PictureInPictureSampleApp.xcodeproj/project.pbxproj

```diff
@@ -608,6 +694,60 @@
 			};
 			name = Release;
 		};
+		3394BB67224BDA3D004CCA1F /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				BUNDLE_LOADER = "$(TEST_HOST)";
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_STYLE = Automatic;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEVELOPMENT_TEAM = 556Y9XWD3N;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				INFOPLIST_FILE = PictureInPictureSampleAppTests/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
```

Comment:
I think we support iOS starting from 9.0

Reply:
Done